### DISCUSSION TITLE
Kata 4

### DISCUSSION THREAD

**Discussion by lunasaurio** [LaunchX-InnovaccionVirtual/CursoIntroPython#28] [Q&A] [answered]
Que tal equipo, tengo la duda de la Kata 4 por que en las palabras clave nos dice que pongamos distance, pero el texto está incompleto como pueden ver
![image](https://user-images.githubusercontent.com/93229275/153097022-7286ebf6-6ed0-4651-a8b6-e2ac2418e8ad.png)
no quisiera limitarme a decir que no esta pero copiando y pegando de todas formas no sale, alguna sugerencia o respuesta?

**Comment by rickygzz** [comment]
Hola,

el texto debe decir lo siguiente

text = """Interesting facts about the Moon. The Moon is Earth's only satellite. There are several interesting facts about the Moon and how it affects life here on Earth. 
On average, the Moon moves 4cm away from the Earth every year. This yearly drift is not significant enough to cause immediate effects on Earth. The highest daylight temperature of the Moon is 127 C."""

Lo recomendable es descargar el archivo y abrirlo directamente en Visual Studio Code, ya que por alguna razón en GitHub no muestra la barra de desplazamiento y el texto se oculta.

Saludos colega,

**Reply by lunasaurio** [reply]
Donde esta la palabra "distance"?

**Reply by amauryram** [reply]
no todas las palabras están, ese es el objetivo del ejercicio, que de las palabras que estén en el arreglo, devuelva la oración que sí tienen dicha palabra. o eso se planea hacer con el ciclo del siguiente ejercicio.

**Reply by carlos-gmx3** [reply]
Yo seleccioné hasta el siguiente renglon. Copy Paste en VS Code y después borré el excedente 
![imagen](https://user-images.githubusercontent.com/99064239/153120332-7ff79862-bf12-4098-8b11-a785eee5779a.png)

**Reply by lunasaurio** [reply]
Muchas gracias por sus aportaciones

**Comment by danielmares32** [comment, marked as answer]
No siempre todas las palabras claves de un tema estan en un texto, porque no siempre tienes a la vista el texto. Por ejemplo que tal que un usuario ingresa el texto. Se sugieren esas palabras pero no te limitan a esas.

**Reply by ramon-ortega** [reply]
"distance" no aparece en el texto, no te preocupes

**Reply by danielmares32** [reply]
Exacto! hahaha
